src/main/java/org/mule/service/http/impl/service/server/grizzly/GrizzlyServerManager.java

```diff
@@ -308,7 +308,7 @@ private HttpServerFilter createHttpServerFilter(int connectionIdleTimeout, boole
     }
     HttpServerFilter httpServerFilter =
         new HttpServerFilter(true, retrieveMaximumHeaderSectionSize(), ka, delayedExecutor);
-    httpServerFilter.getMonitoringConfig().addProbes(new HttpMessageLogger(LISTENER, currentThread().getContextClassLoader()));
+    httpServerFilter.getMonitoringConfig().addProbes(new HttpMessageLogger(LISTENER, identifier.getName(), currentThread().getContextClassLoader()));
```

Comment:
Listeners tend to have the same name across apps, can we guarantee you'll get the desired logs in that case?

Reply:
log4j configuration is per application, so even if they have the same name, they need to configure it on each of them.

Reply:
I meant whether you had tried it, we’ve had issues in the past because of the inbound connections occurring at the container/service level.

Reply:
Tested it with two different applications with the same configuration names and the changes only affects the application where it is defined.